test/keda-reconciler-tests.sh

```diff
@@ -41,4 +41,6 @@ header "Running tests"
 
 go_test_e2e -timeout=1h -run=KafkaSource ./test/e2e_new/... || fail_test "E2E (new) suite failed"
 
+go_test_e2e -tags=sacura -timeout=40m ./test/e2e/... || fail_test "E2E sacura tests failed"
```

Comment:
Can you verify that this is not in LOCAL_DEVELOPMENT before running these tests? We should probably also still have the `-run=KafkaSource` flag set for these tests.

To check the LOCAL_DEVELOPMENT, see https://github.com/knative-extensions/eventing-kafka-broker/blob/315e260db7a27c0e899d6b2a34eacc8e5a06abf3/test/reconciler-tests.sh#L73-L75